internal/resourcepolicies/resource_policies.go

```diff
@@ -0,0 +1,116 @@
+package resourcepolicies
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Action defined as one action for a specific way of backup
+type Action struct {
+	// Type defined specific type of action, currently only support 'skip'
+	Type string `yaml:"type"`
+	// Parameters defined map of parameters when executing a specific action
+	// +optional
+	// +nullable
+	Parameters map[string]interface{} `yaml:"parameters,omitempty"`
+}
+
+// VolumePolicy defined policy to conditions to match Volumes and related action to handle matched Volumes
+type VolumePolicy struct {
+	// Conditions defined list of conditions to match Volumes
+	Conditions map[string]interface{} `yaml:"conditions"`
+	Action     Action                 `yaml:"action"`
+}
+
+// ResourcePolicies currently defined slice of volume policies to handle backup
+type ResourcePolicies struct {
+	Version        string         `yaml:"version"`
+	VolumePolicies []VolumePolicy `yaml:"volumePolicies"`
+	// we may support other resource policies in the future, and they could be added separately
+	// OtherResourcePolicies: []OtherResourcePolicy
+}
+
+type resourcePolicyMatcher interface {
+	Match(res interface{}, actionType string) bool
+}
+
+type volumePolicyMatcherImpl struct {
+	policy      *VolumePolicy
+	matcherFunc func(volume interface{}, actionType string) bool
+}
+
+func (v *volumePolicyMatcherImpl) Match(volume interface{}, actionType string) bool {
+	return v.matcherFunc(volume, actionType)
+}
+
+// Match interface is implemented by VolumePolicy
+func (policy *VolumePolicy) Match(volume interface{}, actionType string) bool {
```

Comment:
Could you reorganize the code and put the related definition in the same go file? Currently, the related definition of `VolumePolicy` in both `resource_policies.go` and `volume_policies.go`